TalkingLeaves.glyphsPlugin/Contents/Resources/TalkingLeaves.py

```diff
@@ -318,7 +315,7 @@ def addDottedCircle(c):
           items.append({
             'ISO': langCode,
             'Language': lang.get('preferred_name', lang['name']),
-            'L1 Speakers': langYaml.get('speakers', -1),
+            'L1 Speakers': lang.get('speakers', -1),
```

Comment:
Since the returned `hyperglot.language.Language` object is just a convenience wrapper around the yaml dict, you can use it in place of a separate dict.

Reply:
The problem is the convenience wrapper adds `speakers: 0` to languages that don't have a `speakers` property, so we can no longer distinguish between languages that actually have zero L1 speakers and languages where we simply don't have data about L1 speakers.

Previously, we could distinguish between them like this:
<img width="523" alt="image" src="https://github.com/justinpenner/TalkingLeaves/assets/46039243/4d431c0f-99bf-4f17-a444-6a0ed2b8b11a">

Is there a way to do this with `hyperglot.language.Language` or should I open an issue about that in the HG repo?

Reply:
Ah, I see how "no data" is useful in this context. The hyperglot CLI has a the option to sort by speakers, so `None` values are a problem there, alas the default of `0`.

I suppose the `Language` object should follow other objects and have _attributes_ for calculated things, and retain the original yaml as the dict (and use the calculated value for the sorting). I'll update this in a future release. For now and here, I'll revert the suggested change.